tr/news/_posts/2019-10-02-ruby-2-4-9-released.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: news_post
+title: "Ruby 2.4.9 Yayınlandı"
+author: "usa"
+translator: "İsmail Arılık"
+date: 2019-10-02 09:00:00 +0000
+lang: tr
+---
+
+Ruby 2.4.9 yayınlandı.
+
+Bu sürüm, 2.4.8'in yeniden paketlenmiş halidir çünkü önceki Ruby 2.4.8 yayın tar arşivi kurulmuyordu.
+(Ayrıntılar için [[Hata #16197]](https://bugs.ruby-lang.org/issues/16197) etiketine bakın.)
+2.4.8 ile 2.4.9 arasında sürüm numaraları dışında önemli bir fark yok.
+
+Ruby 2.4 artık güvenlik sürdürme evresinde, Mart 2020'nin sonuna kadar.
+Bu tarihten sonra, Ruby 2.4'ün sürdürülmesi sona erecek.
+2.6 ya da 2.5 gibi daha yeni Ruby sürümlerine göç etmeyi planlamaya başlamanızı tavsiye ederiz.
```

Comment:
`goc etmek` dogru mu bilemedim

Reply:
`migrate`'in anlamı bu değil mi? Başka ne olabilir?

Reply:
`gecmek, yukseltmek` denebilir. `migrate` goc etmek olabilir ama yazilimda kullanmak bana garip geliyor. gene de sen bilirsin.